house/people.go

```diff
@@ -3,54 +3,77 @@ package house
 import (
 	"context"
 	"fmt"
-	"github.com/google/uuid"
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promauto"
+	pb "github.com/beaujr/nmap_prometheus/proto"
+	etcdv3 "github.com/ozonru/etcd/v3/clientv3"
 	"gopkg.in/yaml.v2"
 	"log"
-	"time"
 )
 
-type people struct {
-	Id      string   `json:"id",yaml:"id"`
-	Devices []string `json:"devices",yaml:"devices"`
-	Name    string   `json:"name",yaml:"name"`
+var peoplePrefix = "/people/"
+
+// Notifier represents and interface for notification sending
+type PeopleManager interface {
+	Get() ([]*pb.Person, error)
+	CreateFromDevices(devices []*pb.Devices, name string) error
+	Create(ids []string, name string) error
+}
+
+// NewNotifier returns a new Notifier
```

Comment:
warning: comment on exported function NewPeopleManager should be of the form "NewPeopleManager ..." (golint)